tests/baselines/reference/privateNameES5Ban(target=es3).errors.txt

```diff
@@ -18,23 +18,23 @@ tests/cases/conformance/classes/members/privateNames/privateNameES5Ban.ts(10,5):
         ~~~~~~~
 !!! error TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.
         static #sField = "hello world";
-        ~~~~~~
-!!! error TS18019: 'static' modifier cannot be used with a private identifier.
+               ~~~~~~~
+!!! error TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.

```

Comment:
I guess technically private static emit can be implemented in earlier versions because it doesn't use `WeakSet` or `WeakMap` but we wan't to have a consistent minimum target?

Reply:
Yes, you are right. I think it would cause more confusion than benefit.